File: guess.py
import random
number = random.randint(1,100)
print('------猜数字游戏！-----')
guess = 0
count = 0
while guess != number:
    temp = input('猜数字游戏开始，请输入数字：')
    guess = int(temp)
    count+=1
    if guess == number:
        print('恭喜，您猜对了！')
        print('您总共用了',count,'次')
        if count < 4:
            print('你太聪明了！')
            break
        if count < 11:
            print('还不错哦')
            break
        if count > 10:
            print('不太行啊')
            break
    if guess > number:
        print('您输入的数字大了！')
    if guess < number:
        print('您输入的数字小了！')
    
    

File: Guess.java
package week4;
import java.util.Random;
import java.util.Scanner;
public class Guess {
	public static void main(String[] args) {
		int randomNumber=new Random().nextInt(10);
	    System.out.println("随机数已生成！");
	    System.out.println("----请输入您猜的数字：----");
		Scanner sc=new Scanner(System.in);
		int enterNumber=sc.nextInt();
		int count=1;
		while(enterNumber!=randomNumber) {
			if(enterNumber>randomNumber) {
				System.out.println("sorry，您猜大了！");
			}else {
				System.out.println("sorry，您猜小了！");
			}
			System.out.println("----请输入您猜的数字：----");
			enterNumber=sc.nextInt();
			count++;
		}
		System.out.println("恭喜您，答对了！");
	    sc.close();
	    System.out.println("您一共用了"+count+"次");
	    if(count==1) {
	    	System.out.println("真厉害！");
	    }else if(count<=3) {
	    	System.out.println("还不错哦！");
	    }else {
	    	System.out.println("还需努力！");
	    }
	}
}
